rudder-core/src/main/scala/com/normation/rudder/services/policies/InterpolatedValueCompiler.scala

```diff
@@ -253,31 +310,29 @@ class InterpolatedValueCompilerImpl extends RegexParsers with InterpolatedValueC
   //exactly quote s - no regex authorized in
   def id(s: String) = ("""(?iu)\Q""" + s + """\E""").r
 
-  def all: Parser[List[Token]] = (rep1(interpol | plainString) | emptyVar)
+  def all: Parser[List[Token]] = (rep1(interpol | nodeProp | plainString) | emptyVar)
 
   //empty string is a special case that must be look appart from plain string.
   //also parse full blank string, because no need to look for more case for them.
-  def emptyVar: Parser[List[CharSeq]] = """(?iums)(\s)*""".r ^^ { x => List(CharSeq(x)) }
+  def emptyVar   : Parser[List[CharSeq]] = """(?iums)(\s)*""".r ^^ { x => List(CharSeq(x)) }
 
-  def plainString: Parser[CharSeq] = ("""(?iums)((?!\Q${rudder.\E).)+""").r  ^^ { CharSeq(_) }
+  // plain string must not match our identifier, ${rudder.* and ${node.properties.*}
+  def plainString: Parser[CharSeq] = ("""(?iums)((?!(\Q${rudder.\E|\Q${node.properties\E)).)+""").r  ^^ { CharSeq(_) }
 
   //identifier for step in the path or param names
-  def propId: Parser[String] = """[\-_a-zA-Z0-9]+""".r
+  def propId     : Parser[String] = """[\-_a-zA-Z0-9]+""".r
```

Comment:
Greedy is perhaps needed, thanks. For the multiline, I think we must forbid it in an identifier itself. It must be authorized between token (and I will add a test to be sure of that), but in one, it seems strange. Have you an use case in mind ?